Sources/BraintreeVisaCheckout/BTVisaCheckoutNonce.swift

```diff
@@ -0,0 +1,55 @@
+import Foundation
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+@objcMembers public class BTVisaCheckoutNonce: BTPaymentMethodNonce {
+
+    // Last two digits of the user's underlying card, intended for display purposes.
+    public let lastTwo: String
+    
+    // Type of this card (e.g. Visa, MasterCard, American Express)
+    public let cardType: String
+    
+    // The user's billing address.
+    public let billingAddress: BTVisaCheckoutAddress
+    
+    // The user's shipping address.
+    public let shippingAddress: BTVisaCheckoutAddress
+    
+    // The user's data.
+    public let userData: BTVisaCheckoutUserData
+    
+    // The Call ID from the VisaPaymentSummary.
+    public let callID: String
+    
+    // The BIN data for the card number associated with [VisaCheckoutNonce]
+    public let binData: BTBinData
+
+    init?(json: BTJSON) {
+
+        let json = json["visaCheckoutCards"].isArray ? json["visaCheckoutCards"][0] : json
+        
+        guard
+            let lastTwo = json["details"]["lastTwo"].asString(),
+            let cardType = json["details"]["cardType"].asString(),
+            let nonce = json["nonce"].asString(),
+            let type = json["type"].asString(),
+            let callID = json["callId"].asString()
+        else {
+            return nil
+        }
```

Comment:
It looks like in the rest of the SDK we just return nil of the nonce is not returned. Do we want to do the same here?